src/inputs/input.tsx

```diff
@@ -61,11 +62,12 @@ export default class Input extends React.Component<IInputProps, {}> {
         const hasError = this.props.fieldErrors && this.props.fieldErrors.length;
         const className = `${this.props.className} ${hasError ? 'error' : ''}`;
 
-        let props = this.props;
-        delete props.name;
-        delete props.fieldErrors;
-        delete props.validateOnBlur;
-        delete props.validators;
+        const props = omit([
```

Comment:
Before you deleted `props.name` and with the change you omit `props.error` instead. Was this intentional?